addon/components/one-way-checkbox.js

```diff
@@ -20,8 +20,9 @@ const OneWayCheckboxComponent = Component.extend(DynamicAttributeBindings, {
     'value'
   ],
 
-  click(event) {
-    invokeAction(this, 'update', this.readDOMAttr('checked'), event);
+  didInsertElement() {
```

Comment:
@rwjblue There's a test that assets that if you call `.stopPropagation` on an action the event doesn't bubble. I started to fail when I changed the dispatcher.

This is is: https://github.com/DockYard/ember-one-way-controls/blob/master/tests/integration/components/one-way-checkbox-test.js#L91-L113